WORKSPACE

```diff
@@ -25,14 +25,12 @@ rules_cc_internal_deps()
 load("//:internal_setup.bzl", "rules_cc_internal_setup")
 rules_cc_internal_setup()
 
-# We're pinning to a commit because this project does not have a recent release.
-# Nothing special about this commit, though.
 http_archive(
     name = "com_google_googletest",
-    sha256 = "0fb00ff413f6b9b80ccee44a374ca7a18af7315aea72a43c62f2acd1ca74e9b5",
-    strip_prefix = "googletest-f13bbe2992d188e834339abe6f715b2b2f840a77",
+    sha256 = "4d7cd95cdb0ef420eed163696a906cccd70964801ea611b2020f31177432c27d",
```

Comment:
Isn't `1.10.x` the branch, hence doesn't guarantee a stable hash?

Reply:
Oh you're right, I copypasted the wrong link! Updated the PR now.